include/circt/Support/LoweringOptionsParser.h

```diff
@@ -48,7 +48,7 @@ struct LoweringOptionsOption
                 "locationInfoStyle={plain,wrapInAtSquareBracket,none}, "
                 "disallowPortDeclSharing, printDebugInfo, "
                 "disallowExpressionInliningInPorts, disallowMuxInlining, "
-                "emitWireInPort, emitBindComments"),
+                "emitWireInPort, emitBindComments, caseInsensitiveKeywords"),
```

Comment:
(Looks like `omitVersionComment` is missing here! Unrelated just wanted to note somewhere :+1: )